daemons/controld/controld_remote_ra.c

```diff
@@ -324,7 +323,8 @@ check_remote_node_state(const remote_ra_cmd_t *cmd)
         remote_ra_data_t *ra_data = lrm_state? lrm_state->remote_ra_data : NULL;
 
         if (ra_data) {
-            if (ra_data->migrate_status != takeover_complete) {
+            if (!pcmk_is_set(ra_data->status, expect_takeover) &&
+                !pcmk_is_set(ra_data->status, takeover_complete)) {
```

Comment:
It was a little unclear to me whether this needs to check both,or whether it's good enoguh to just check that takeover_complete is not set.  Note that in the original code, expect_takeover was 1.

Reply:
just takeover_complete (expect_takeover should never be set when takeover_complete is anyway)